articles/quickstart/webapp/any/01-login.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to integrate Auth0 authentication into any conventional server-based web application.
+budicon: 
+---
+
+This guide explains in general terms how to integrate Auth0 authentication into a conventional server-based web app serving static HTML pages to the user. 
+
+## Overview
+
+This is how user authentiation happens between a conventional server-side web application and the Auth0 service. 
+
+1. Your application redirects the user's browser to Auth0 `/authorize` endpoint, with a *redirect URI* in the query string.
+2. The user logs in, using the Auth0 login form on that page.
+3. Auth0 authenticates the user. 
+4. Auth0 redirects the user to your `redirect_uri`, with an *Authorization Code* in the querystring.
+5. Your application calls the Auth0 /oauth/token endpoint with the Authorization Code, asking to exchange it with an access_token.
+6. Auth0 authenticates the web app, validates the Authorization Code, and responds back with the token.
```

Comment:
/oauth/token does not return just an access_token, and below when talking about what we get, access_tokens are not mentioned

Reply:
I got confused because some of the other documentation doesn't make the distinction between the different tokens and the JSON object clear. The next commit will correct this throughout.